lib/layout/forceDirectedLayout.js

```diff
@@ -173,6 +172,38 @@ var springForce = function(edge) {
   }
 };
 
+var layoutPorts = function(graph) {
+  var verts = graph.get('vertices');
+  var maxPortsPerSide = 6;
+  for(var i = 0; i < verts.length; i++) {
+    var ins = verts[i].get('inputs');
+    var outs = verts[i].get('outputs');
+    for(var j = 0; j < ins.length; j++) {
+      var posPercentage = 12 + 15 * (j % maxPortsPerSide);
+      if(j < maxPortsPerSide) {
```

Comment:
I'm looking at this look. It doesn't look like any of the contents of the loop with execute when `j < maxPortsPerSide` but the loop will keep looping. Why not stop the loop?